report_format_option/models/ir_actions_report.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2024 Quartile Limited
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
+
+from odoo import api, fields, models
+
+
+class IrActionReport(models.Model):
+    _inherit = "ir.actions.report"
+
+    encoding = fields.Char(
+        help="Encoding to be applied to the generated CSV file. e.g. cp932"
+    )
+    encode_error_handling = fields.Selection(
+        selection=[("ignore", "Ignore"), ("replace", "Replace")],
+        help="If nothing is selected, CSV export will fail with an error message when "
+        "there is a character that fail to be encoded.",
+    )
+    show_encoding = fields.Boolean(
+        compute="_compute_show_encoding",
+        help="Technical field to control the visibility of the encoding field.",
+    )
+    line_ending = fields.Selection(
+        [("LF", "LF (\\n)"), ("CRLF", "CRLF (\\r\\n)"), ("CR", "CR (\\r)")],
+        default="LF",
+        help="Select the type of line ending.",
```

Comment:
```suggestion
        [("crlf", "CRLF (\\r\\n)"), ("cr", "CR (\\r)")],
        help="Select the type of line ending in case the report needs to be output with other line ending than 'LF'.",
```